optuna/pruners/_base.py

```diff
@@ -1,11 +1,34 @@
 import abc
+from typing import Dict
 
 import optuna
 
 
 class BasePruner(object, metaclass=abc.ABCMeta):
     """Base class for pruners."""
 
+    SPECIAL_KEYWORDS: Dict[str, str] = {}
+
+    def __repr__(self) -> str:
+        import inspect
```

Comment:
maybe we can move this import line to the top?

Reply:
d7ceae7 removes the importing of `inspect`!